src/test/testUtil.ts

```diff
@@ -207,3 +209,71 @@ export const assertTelemetryCurried =
     <K extends MetricName>(name: K) =>
     (expected: TelemetryMetric<K>) =>
         assertTelemetry(name, expected)
+
+/**
+ * Waits for _any_ active text editor to appear and have the desired contents.
+ * This is important since there may be delays between showing a new document and
+ * updates to the `activeTextEditor` field.
+ *
+ * Assumes that only a single document will be edited while polling. The contents of
+ * the document must match exactly to the text editor at some point, otherwise this
+ * function will timeout.
+ */
+export async function assertTextEditorContains(contents: string): Promise<void | never> {
+    const editor = await waitUntil(
+        async () => {
+            if (vscode.window.activeTextEditor?.document.getText() === contents) {
+                return vscode.window.activeTextEditor
+            }
+        },
+        { interval: 5 }
+    )
+
+    if (!vscode.window.activeTextEditor) {
+        throw new Error('No active text editor found')
+    }
+    if (!editor) {
+        assert.strictEqual(vscode.window.activeTextEditor.document.getText(), contents)
+    }
+}
+
+/**
+ * Executes the close all editors command and waits for all visible editors to disappear
+ */
+export async function closeAllEditors(): Promise<unknown> {
+    if ((await vscode.commands.getCommands()).includes('openEditors.closeAll')) {
+        if (vscode.version.startsWith('1.44') && !isReleaseVersion()) {
+            throw Error('openEditors.closeAll is available in min version, remove me!')
```

Comment:
Just so we know when the other code is redundant. I moved the function to be `test` but added the `isReleaseVersion` guard just in case...

Reply:
> Also we should define what our min-ver is somewhere so it's not hard-coded. I guess just derive it from package.json

Yeah I thought about that in previous PRs but for many cases we don't really want the skip to auto-bump whenever we bump package.json...